libs/SalesforceAnalytics/SalesforceAnalytics/Classes/Logger/SFSDKLogger.h

```diff
@@ -144,4 +197,103 @@
  */
 - (void)log:(nonnull Class)class level:(DDLogLevel)level format:(nonnull NSString *)format, ...;
 
+/**
```

Comment:
You're right. My bad. That's why I shouldn't be doing code reviews at 6 A.M. with no sleep the previous night.

Looks good to me. 👍